content/about/who-we-are.md

```diff
@@ -19,22 +19,28 @@ David Newbury, is the Head of Software at Getty …
 Matthew Hrudka, is a Software Architect at Getty …
 
 **Erin Cecele Dunigan** ([@Erin-Cecele](https://github.com/Erin-Cecele)), community manager<br />
-Erin Cecele Dunigan, is the Quire Community Manager at Getty, where she is cultivating a diverse community of Quire users and contributors. She works closely with the Project Manager and Getty developers to communicate user needs and develop helpful tools and resources to aid the community in Quire adoption. She has worked in art book publishing for over a decade, including positions at ARTBOOK | D.A.P. and Chronicle Books. She is also a co-founder of the San Francisco Art Book Fair. 
+Erin Cecele Dunigan, is the Quire Community Manager at Getty, where she is cultivating a diverse community of Quire users and contributors. She works closely with the Project Manager and Getty developers to communicate user needs and develop helpful tools and resources to aid the community in Quire adoption. She has worked in art book publishing for over a decade, including positions at ARTBOOK | D.A.P. and Chronicle Books. She is also a co-founder of the San Francisco Art Book Fair.
 
 ## Contributors
 
-Thank you to all our growing list of contributors who have made Quire what it is, and are working on what it will be!
+Thank you to all our growing list of contributors who have made Quire what it is, and are working on what it will be! They’ve contributed code, written and edited documentation, run events, and blogged and spoken about Quire.
 
+- [@aheltonsjma](https://github.com/aheltonsjma)
 - [@anderspollack](https://github.com/anderspollack)
+- [@antoinentl](https://github.com/antoinentl)
 - [@daniel-keller](https://github.com/daniel-keller)
 - [@egardner](https://github.com/egardner)
 - [@fchasen](https://github.com/fchasen)
 - [@ggarcia0596](https://github.com/ggarcia0596)
 - [@kalvarenga](https://github.com/kalvarenga)
+- [@kristhayer11](https://github.com/kristhayer11)
 - [@mandrijauskas](https://github.com/mandrijauskas)
+- [@MillsArtMuseum](https://github.com/MillsArtMuseum)
 - [@mbelhu](https://github.com/mbelhu)
 - [@naeluh](https://github.com/naeluh)
+- [@nancyum](https://github.com/nancyum)
 - [@oncletom](https://github.com/oncletom)
+- [@swambold1](https://github.com/swambold1)
 - [@victoriabarry](https://github.com/victoriabarry)
 
```

Comment:
When we update contributors, will we need to do it separately in the README.md and in Who We Are? Any way for them to speak to each other?

Reply:
Yes, we'd need to do it in both places, and unfortunately, because the README lives outside the content folder, there's no easy way to update both from a single location. The best we could probably do is create a Quire template to generate a README file that we could then manually move to the right location. But would probably take more work than it's worth at this point.

Reply:
Not a problem. As long as I have a checklist of places that require monthly(ish) updates then we should be ok.